redaxo/src/addons/debug/boot.php

```diff
@@ -8,6 +8,43 @@
 if (rex::isBackend() && 'debug' === rex_request::get('page')) {
     $index = file_get_contents(rex_addon::get('debug')->getAssetsPath('clockwork/index.html'));
 
+    $editor = rex::getProperty('editor');
+    $editorBasepath = rex::getProperty('editor_basepath');
```

Comment:
Ich denke für die 2 properties sollten wir einen getter in der rex_editor klasse mache